be/src/olap/lru_cache.cpp

```diff
@@ -491,11 +488,13 @@ Cache::Handle* ShardedLRUCache::lookup(const CacheKey& key) {
 }
 
 void ShardedLRUCache::release(Handle* handle) {
+    SCOPED_SWITCH_THREAD_LOCAL_MEM_TRACKER(_mem_tracker);
```

Comment:
Can we add a metric counter for this SWITCH operation?
I want to know how many SWITCH may be called.

Reply:
ok, i try to add

Reply:
fix